src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return days * cost - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return days * cost - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
According to the checklist, you should avoid using `else` after an `if` statement that contains a `return`. Simply use another `if` statement for the next condition. This helps to flatten the structure of the code and makes it more readable.